server/settings/main.py

```diff
@@ -76,6 +83,49 @@ def _ocio_built_in_paths():
     ]
 
 
+class FallbackProductModel(BaseSettingsModel):
+    _layout = "expanded"
+    type: str = SettingsField(
```

Comment:
I hightly recommend to rename the attribute as `type` is built-in python function, e.g. `fallback_type`.

Reply:
done